packages/xo-server/src/api/proxy.mjs

```diff
@@ -183,6 +183,18 @@ getApplianceUpdaterState.params = {
   },
 }
 
+export function getServerVersion({ id }) {
+  const versionInfo = this.callProxyMethod(id, 'system.getServerVersion')
+  return versionInfo
+}
+
+getServerVersion.permission = 'admin'
+getServerVersion.params = {
+  id: {
+    type: 'string',
+  },
+}
+
```

Comment:
@MathieuRA What do you think of integrating this info directly in the proxy objects? (`proxies.getAll`)

Reply:
Yes, this might be a good idea and simplify the code on the UI side

Reply:
It will make it simpler to expose in the REST API as well.